ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.js

```diff
@@ -80,13 +84,13 @@ const ConfirmAddSuggestedToken = () => {
   const tokens = useSelector(getTokens);
   const trackEvent = useContext(MetaMetricsContext);
 
-  const knownTokenActionableMessage = useMemo(() => {
+  const knownTokenBannerAlert = useMemo(() => {
     return (
       hasDuplicateAddress(suggestedTokens, tokens) && (
-        <ActionableMessage
-          message={t('knownTokenWarning', [
+        <BannerAlert severity={Severity.Warning}>
+          {t('knownTokenWarning', [
             <Button
-              type="link"
+              variant={BUTTON_VARIANT.LINK}
               key="confirm-add-suggested-token-duplicate-warning"
               className="confirm-add-suggested-token__link"
```

Comment:
Can we remove this class and removed the associated styles 
```
  &__link {
    @include H7;

    display: inline;
    color: var(--color-primary-default);
    padding-left: 0;
  }
``` 

from `ui/pages/confirm-add-suggested-token/index.scss` 

We want the link here to be the same size as the body text

<img width="679" alt="Screenshot 2023-08-28 at 5 02 58 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/087f3adf-4beb-4c61-b1cf-306e45cd3557">